src/main/java/com/opentable/db/postgres/embedded/EmbeddedPostgres.java

```diff
@@ -187,7 +187,7 @@ public int getPort()
 
     private static int detectPort() throws IOException
     {
-        try (final ServerSocket socket = new ServerSocket(0)) {
+        try (ServerSocket socket = new ServerSocket(0)) {
```

Comment:
Not that I care much but a lot of dropping of the final keyword

Reply:
It's a PMD "fix".  `final` is redundant for variables declared in a try-with-resources since they are necessarily `final`.